pkg/devfile/header.go

```diff
@@ -81,4 +81,12 @@ type DevfileMetadata struct {
 	// Optional devfile website
 	// +optional
 	Website string `json:"website,omitempty"`
+
+	// Optional devfile provider information
+	// +optional
+	Provider string `json:"provider,omitempty"`
+
+	// Optional link to a page that provides support information
+	// +optional
+	SupportURL string `json:"supportURL,omitempty"`
```

Comment:
To match other instances of URL in the yaml spec, I think the json tag should be `supportUrl` (c.f. `registryUrl`)

Reply:
updated